components/brave_wallet_ui/page/screens/backup-wallet/backup-enter-password/backup-enter-password.tsx

```diff
@@ -97,54 +93,46 @@ export const BackupEnterPassword: React.FC = () => {
     setPassword(value)
   }
 
-  const goBackUrl =
-    history.action === 'POP' ? WalletRoutes.PortfolioAssets : undefined
-
   // render
   return (
-    <CenteredPageLayout>
-      <MainWrapper>
-        <StyledWrapper>
-          <StepsNavigation
-            currentStep={WalletRoutes.OnboardingExplainRecoveryPhrase}
-            steps={[]}
-            goBackUrl={goBackUrl}
-          />
-
-          <div>
-            <Title>{getLocale('braveWalletEnterAPasswordToContinue')}</Title>
-            <Description>
-              {getLocale('braveWalletEnterYourPasswordToStartBackup')}
-            </Description>
-
-            <VerticalSpacer space={56} />
-
-            <PasswordInput
-              placeholder={getLocale('braveWalletEnterYourPassword')}
-              label={getLocale('braveWalletInputLabelPassword')}
-              onChange={onPasswordChange}
-              onKeyDown={handlePasswordKeyDown}
-              hasError={!!password && !isCorrectPassword}
-              value={password}
-              error={getLocale('braveWalletLockScreenError')}
-              autoFocus
-              name='password'
-            />
-          </div>
-
-          <VerticalSpacer space={194} />
-
-          <NextButtonRow>
-            <NavButton
-              buttonType='primary'
-              text={getLocale('braveWalletButtonContinue')}
-              disabled={!password || !isCorrectPassword}
-              onSubmit={onSubmit}
-            />
-          </NextButtonRow>
-        </StyledWrapper>
-      </MainWrapper>
-    </CenteredPageLayout>
+    <OnboardingContentLayout
+      showBackButton
```

Comment:
What does the `back` button do in this instance?

Reply:
Navigates the user back to portfolio